Tasks/XamarinAndroidV1/package.json

```diff
@@ -18,6 +18,7 @@
   "homepage": "https://github.com/microsoft/vsts-tasks#readme",
   "dependencies": {
     "@types/node": "^6.0.101",
+    "@types/q": "^1.5.1",
```

Comment:
Is this needed?

Reply:
Yeah, without it, the compiler errors because I enabled `noImplicitAny` -- it's in the declaration file (.d.ts) for the task lib.

We could also disable type checking on the declaration files (with `skipLibCheck`) but I think we still want that.